src/plugins/posix/io_uring_io_queue.cpp

```
@@ -51,20 +54,40 @@ class nixlPosixIOQueueUring : public nixlPosixIOQueueImpl<nixlPosixIoUringIO> {
             void *ctx) override;
     virtual nixl_status_t
     poll(void) override;
+    virtual nixl_status_t
+    cancel(void *ctx) override;
+
+    virtual bool
+    hasPendingCleanup(void) const override {
```

Cppcheck (2.21.0)

[style] The function 'hasPendingCleanup' is never used.

(unusedFunction)